pkg/kubecfg/pack.go

```diff
@@ -201,6 +205,18 @@ func (c PackCmd) pushOCIBundle(ctx context.Context, ref string, bodyBlob []byte,
 	return nil
 }
 
+func getSourceRevision(rootFile string) string {
+	repo, err := git.PlainOpenWithOptions(rootFile, &git.PlainOpenOptions{DetectDotGit: true})
+	if err != nil {
```

Comment:
A debug log in case if error here would be nice. But it's not blocking for this PR